EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py

```diff
@@ -4,7 +4,10 @@
     "L1TDigiToRaw",
     Setup = cms.string("stage2::GTSetup"),
 #    TowerInputLabel = cms.InputTag("simCaloStage2Digis"),
-    InputLabel = cms.InputTag("simGtStage2Digis"),
+    GtInputLabel = cms.InputTag("simGtStage2Digis"),
```

Comment:
@mulhearn - these aren't available for data... whats the timescale for making the real digis?